### DISCUSSION TITLE
Weird new player behaviour

### DISCUSSION THREAD

**Discussion by Muldoni** [smplayer-dev/smplayer#888] [Q&A] [answered]
Starting a few days ago, everytime I start playing a new local video file, the player will play the file normally for the first one or two seconds (with audio), then it suddenly starts to fast forward the video for between 3-7 seconds realtime (the video itself will forward to somewhere between 30 seconds and 2 minutes of playtime), before it suddenly slows back down and continues to play the file in normal speed (but now without audio). It will then repeat this behavior in irregular intervals.

This happens on every file I tried, and with every file format, I have tried mp4, wmv and mkv.
This behavior started last week on a previous version of SMPlayer that used to run fine. I have since updated to the newest version and also resinstalled from scratch but the error remains. All these files will play normally on other media players. 

Does anybody have an idea what the issue might be?

**Comment by Raza-0007** [comment, marked as answer]
Try resetting smplayer to default by deleting the `smplayer.ini` file

**Reply by Muldoni** [reply]
Awesome, that did it! Thanks a lot!
